### ISSUE TITLE
GitHub page in German

### ISSUE THREAD

**Issue by warm-coolguy** [Dataport/polar#102]
Our GitHub page should have a german translation for stakeholders not used to reading English. We may try to build in a language switcher and i18n, but might as well just offer a `.de.html` next to the english version as a quick solution to go with English as our lingua franca and use the german documents in parallel.

**Comment by warm-coolguy** [comment]
@dopenguin I propose we add WONTFIX and close this until further demand.

**Comment by warm-coolguy** [comment]
📞 @dopenguin
